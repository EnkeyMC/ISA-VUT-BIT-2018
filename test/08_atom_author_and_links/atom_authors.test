NAME=Basic Atom author parsing
ARGS=http://www.mocky.io/v2/5bf29a9b2f00008a00cfa190 -a
RC=0
CHECK_OUTPUT=1
