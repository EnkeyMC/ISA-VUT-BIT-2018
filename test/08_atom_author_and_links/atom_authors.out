*** xkcd.com ***
I'm a Car
Autor: Joe Done

Carnot Cycle
Autor: Black Betty

Barnard's Star
Autor: John Doe; Jane Doe

Tectonics Game
Autor: John Doe; Jane Doe
